src/test/java/edu/cornell/kfs/rass/batch/xml/fixture/RassXmlAwardEntryFixture.java

```diff
@@ -0,0 +1,104 @@
+package edu.cornell.kfs.rass.batch.xml.fixture;
+
+import java.util.Date;
+
+import org.apache.commons.lang3.StringUtils;
+import org.joda.time.format.DateTimeFormatter;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+import edu.cornell.kfs.rass.batch.xml.RassXmlAwardEntry;
+import edu.cornell.kfs.rass.batch.xml.RassXmlDocumentWrapperMarshalTest;
+
+public enum RassXmlAwardEntryFixture {
+    FIRST("141414", "OS", "2345", "First Example Project", "2017-12-31", null, new KualiDecimal(5300000.000), new KualiDecimal(700000.000), new KualiDecimal(6000000.000),
```

Comment:
If you want to make the enum constant setup a little cleaner, you could update the constructor to take in doubles instead of KualiDecimals, and then convert them to KualiDecimals within the constructor.

Reply:
changed